resources.go

```diff
@@ -282,9 +282,18 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		Resources: map[string]*tfbridge.ResourceInfo{
 			// Active Directory (AAD)
-			"azurerm_azuread_application":                {Tok: azureResource(azureAD, "Application")},
-			"azurerm_azuread_service_principal":          {Tok: azureResource(azureAD, "ServicePrincipal")},
-			"azurerm_azuread_service_principal_password": {Tok: azureResource(azureAD, "ServicePrincipalPassword")},
+			"azurerm_azuread_application": {
+				Tok:                azureResource(azureAD, "Application"),
+				DeprecationMessage: "please use azuread.Applicaton",
```

Comment:
i think we need to expand on this.  I'm not sure how a user woudl get from this message to knowing that they (presumably) need to reference a new package.  This may entail specific deprecation messages per language.  i.e. "The types and functions from the @pulumi/azure/ad module have moved to the @pulumi/azuread package."